terraform/variables.tf

```diff
@@ -5,7 +5,7 @@ variable "region" {
 
 variable "stage" {
   description = "The stage/environment to deploy to. Suggest: `sandbox`, `development`, `staging`, `production`."
-  default     = "development"
+  default     = "nonprod"
```

Comment:
We need to go through all of the docs and code everywhere to make sure we (1) explain and (2) harmonize this new `nonprod` thing. 

It's not obvious what it even means vs. the other stages in https://github.com/FormidableLabs/aws-lambda-serverless-reference/#stages . And, all the examples use `STAGE=sandbox`.

Perhaps, revert all this `nonprod` stuff and integrate in a separate PR so as to not confuse two different meta-level changes?

Reply:
Is this comment addressed? I would expect to see a lot more documentation changes to identify what `nonprod` means now?

Reply:
This was a straggler, removed!